test/cppsuite/test_harness/workload_generator.h

```diff
@@ -0,0 +1,168 @@
+/* Include guard. */
+#ifndef WORKLOAD_GENERATOR_H
+#define WORKLOAD_GENERATOR_H
+
+#include <cstdint>
+#include <map>
+#include <vector>
+
+extern "C" {
+#include "test_util.h"
+}
+
+#include "api_const.h"
+#include "random_generator.h"
```

Comment:
Lets be pedantic and do our includes in the following order:
 - Standard includes, alphabetically sorted
 - extern C includes, alphabetically sorted
 - local includes, alphabetically sorted

Can you fix up the includes in this files and other ones in the project, if they don't match this rule?